app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -180,7 +182,10 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
 
         webView = findViewById(R.id.webview)
         webView.apply {
-            setOnTouchListener { _, _ ->
+            setOnTouchListener { _, motionEvent ->
+                if (motionEvent.pointerCount == 3 && motionEvent.action == MotionEvent.ACTION_POINTER_3_DOWN) {
+                    dispatchKeyEvent(KeyEvent(KeyEvent.ACTION_DOWN, KeyEvent.KEYCODE_E))
```

Comment:
Rather than faking the key presses we should probably send the javascript event on the bus.

Reply:
Is there an actual event for this?I didn't see it when I looked